samples/customABSample/src/main/java/com/inmobi/customabsample/InterstitialCustomABActivity.java

```diff
@@ -0,0 +1,145 @@
+        setContentView(R.layout.activity_interstitial_custom_a_b);
+        mPreloadButton = findViewById(R.id.button_preload_ad);
+        mLoadAdButton = findViewById(R.id.button_load_ad);
+        mShowAdButton = findViewById(R.id.button_show_ad);
+        mPreloadButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (null == mInterstitialAd) {
+                    setupInterstitial();
+                } else {
+                    mInterstitialAd.getPreloadManager().preload();
+                }
+            }
+        });
+        mLoadAdButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                mInterstitialAd.getPreloadManager().load();
+            }
+        });
+        mShowAdButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                mInterstitialAd.show();
+                mPreloadButton.setVisibility(View.VISIBLE);
+                mShowAdButton.setVisibility(View.GONE);
+            }
+        });
+        setupInterstitial();
+    }
+
+    private void setupInterstitial() {
+        mInterstitialAd = new InMobiInterstitial(InterstitialCustomABActivity.this, PlacementId.YOUR_INTERSTITIAL_PLACEMENT_ID,
+                new InterstitialAdEventListener() {
+                    @Override
+                    public void onAdLoadSucceeded(@NonNull InMobiInterstitial inMobiInterstitial,
+                                                  @NonNull AdMetaInfo adMetaInfo) {
+                        Log.d(TAG, "onAdLoadSuccessful");
+                        if (inMobiInterstitial.isReady()) {
+                            mLoadAdButton.setVisibility(View.GONE);
+                            if (mShowAdButton != null) {
+                                mShowAdButton.setVisibility(View.VISIBLE);
+                            }
+                        } else {
+                            Log.d(TAG, "onAdLoadSuccessful inMobiInterstitial not ready");
+                        }
+                    }
+
+                    @Override
+                    public void onAdLoadFailed(InMobiInterstitial inMobiInterstitial, InMobiAdRequestStatus inMobiAdRequestStatus) {
+                        super.onAdLoadFailed(inMobiInterstitial, inMobiAdRequestStatus);
+                        Log.d(TAG, "Unable to load interstitial ad (error message: " +
+                                inMobiAdRequestStatus.getMessage());
+                    }
+
+                    @Override
+                    public void onAdFetchSuccessful(@NonNull InMobiInterstitial inMobiInterstitial, @NonNull AdMetaInfo adMetaInfo) {
+                        super.onAdFetchSuccessful(inMobiInterstitial, adMetaInfo);
+                        Log.d(TAG, "onAdFetchSuccessful");
+                        if (mLoadAdButton != null) {
```

Comment:
Purpose of using custom AB is to get a Bid at right point. please show case how to get the adMetaInfo and bid

Reply:
Added